tasks/plugins.yml

```diff
@@ -25,5 +25,5 @@
   command: >
     java -jar {{ jenkins_jar_location }} -s http://{{ jenkins_hostname }}:{{ jenkins_http_port }}{{ jenkins_url_prefix | default('') }}/ install-plugin {{ item }}
     creates=/var/lib/jenkins/plugins/{{ item }}.jpi
-  with_items: jenkins_plugins
+  with_items: '{{ jenkins_plugins }}'
```

Comment:
Does this break Ansible 1.8.x or 1.9.x? If so, we need to bump the role requirements to the version this was first supported...